app/src/main/res/values/dimens.xml

```diff
@@ -590,5 +590,7 @@
 
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
-
+   
+  <!-- StoryFragment -->
+  <dimen name="story_fragment_padding_bottom">60dp</dimen>
```

Comment:
This should be `160dp` please check again.

Reply:
Yes, I changes it.